tools/crm_simulate.c

```diff
@@ -584,7 +584,7 @@ create_dotfile(pe_working_set_t * data_set, const char *dot_file, gboolean all_a
     FILE *dot_strm = fopen(dot_file, "w");
 
     if (dot_strm == NULL) {
-        g_set_error(error, G_OPTION_ERROR, pcmk_rc2exitc(errno),
+        g_set_error(error, PCMK__EXITC_ERROR, pcmk_rc2exitc(errno),
```

Comment:
An errno is a valid pacemaker rc, so we could just pass it straight as PCMK__RC_ERROR

errnos can vary from OS to OS, but I don't think we'd ever try to pass a GError across hosts ...

Reply:
Fixed.